src/constants.ts

```diff
@@ -11,7 +11,7 @@ export const MIN_WIDTH_SELECTOR_TO_MEASURE = `.${ MIN_WIDTH_CLASS_TO_MEASURE }`;
 export const SCREENSHOT_WIDTH = 1040;
 export const SCREENSHOT_HEIGHT = 1248;
 export const LIMIT_OF_ZIP_SITES_PER_USER = 10;
-export const LIMIT_OF_PROMPTS_PER_USER = 1000;
+export const LIMIT_OF_PROMPTS_PER_USER = 200;
```

Comment:
This is not related to the main purpose of this PR, but it fixes an incorrect change I made in https://github.com/Automattic/studio/pull/1288. 200 is the prompt limit for regular Studio users.

Reply:
Correct. Thanks for the fix.